FusionIIIT/templates/academic_procedures/addCourse.html

```diff
@@ -0,0 +1,10 @@
+{% block title %}Add Courses{% endblock %}
+
+{% block content %}
+  <h2>Login</h2>
+  <form method="post">
+    {% csrf_token %}
+    {{ CourseForm.as_p }}
+    <button type="submit">Login</button>
+  </form>
+{% endblock %}
```

Comment:
@achintMI Why is the template being added?

Reply:
@Samy-33 Actually, we thought of showing student registration form here, just ignore the login mentioned in the code.

Reply:
@achintMI , Ok for now. You can add these files in your local env don't push them on your repository so that nothing conflicts.